android-interop-testing/app/src/main/java/io/grpc/android/integrationtest/InteropTester.java

```diff
@@ -124,39 +110,11 @@ public void onCompleted() {
 
 
   public InteropTester(String testCase,
```

Comment:
How does this PR compile? You didn't change any usages of this constructor.

Similarly, there are no usages of TesterOkHttpChannelBuilder.